whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapExceptionMapper.java

```diff
@@ -69,12 +68,12 @@ public Response toResponse(final Exception exception) {
         }
 
         if (exception instanceof EmptyResultDataAccessException) {
-            return Response.status(HttpServletResponse.SC_NOT_FOUND).entity(createErrorEntity(HttpServletResponse.SC_NOT_FOUND, "not found")).build();
+            return Response.status(HttpServletResponse.SC_NOT_FOUND).entity(createErrorEntity(HttpServletResponse.SC_NOT_FOUND, "404 Not Found")).build();
         }
 
         if (exception instanceof QueryException) {
             if ( ((QueryException) exception).getCompletionInfo() == QueryCompletionInfo.BLOCKED) {
-                return createErrorResponse(Response.Status.TOO_MANY_REQUESTS, exception.getMessage());
```

Comment:
RdapQueryHandler should now throw RdapException instead of QueryException, we can get rid of these unnecessary exceptions

Reply:
we throw this error in ElasticSeachAccountingCallback line 37/39
and this is being called from different sides, not just rdap. I need to put the try catch in the controller layer

Reply:
We have RdapElasticFullTextSearchService for this. RdapElasticFullTextSearchService should catch this exception and turn it into rdapException. RdapExceptionMapper is just for rdap queries so it will have no impact on any other service calls